tests/run_ipolates_tests.sh

```diff
@@ -38,6 +38,11 @@ echo "*** Testing new_grid_order on file in incorrect order."
 touch new_grid_reorder.txt
 diff -w new_grid_reorder.txt new_grid.txt
 
+echo "*** Testing conversion to wmo rot lat-lon grid"
+../wgrib2/wgrib2 test.grb -new_grid_winds earth -new_grid rot-ll:10:-40:0 342:665:0.0625 -20:657:0.0625 \
+   junk.grb
```

Comment:
When creating files in testing, it's a good practice to include the name of the test in the file name. Like "run_ipolates_tests_junk.grb" instead of "junk.grb".

This prevents tests from overwriting each other's test files when parallel execution is used (i.e. "make -j tests").

Also it allows us to quickly locate test files when debugging.